src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -278,6 +278,24 @@ public function getCondition()
                     'label' => $this->translator->trans('Refurbished', [], 'Shop.Theme.Catalog'),
                     'schema_url' => 'https://schema.org/RefurbishedCondition',
                 ];
+            case 'damaged':
+                return [
+                    'type' => 'damaged',
+                    'label' => $this->translator->trans('Damaged', [], 'Shop.Theme.Catalog'),
+                    'schema_url' => 'https://schema.org/DamagedCondition',
+                ];
+            case 'open_package':
+                return [
+                    'type' => 'open_package',
+                    'label' => $this->translator->trans('Opened package', [], 'Shop.Theme.Catalog'),
+                    'schema_url' => 'https://schema.org/NewCondition',
+                ];
+            case 'b_grade':
+                return [
+                    'type' => 'b_grade',
+                    'label' => $this->translator->trans('B grade', [], 'Shop.Theme.Catalog'),
```

Comment:
What about "New with defects". This way we understand the product is new but could have small imperfections.